Dockerfile

```diff
@@ -109,6 +109,9 @@ COPY --from=node-build /usr/src/app/dist ./dist
 COPY --from=node-deps /build/node_modules ./node_modules
 COPY --from=node-deps /build/rootfs-overlay/ /
 
+# Remove default nproc limit for Avahi for it to work in-container
+RUN sed -i "s/rlimit-nproc=3//" /etc/avahi/avahi-daemon.conf
```

Comment:
Since we're not enabling cross compilation in this last step, RUN statements won't work here on ARM builds. Might be better to just COPY a modified avahi-daemon.conf - or modify it in resin-supervisor-base.